docs/announcement.md

```diff
@@ -0,0 +1,116 @@
+We announce both community-ee-base and community-ee-minimal together
```

Comment:
It's best if a MarkDown document starts with a top-level title, and then only uses lower-level titles. So:
```suggestion
# Announcing the community-ee-* execution environments

We announce both community-ee-base and community-ee-minimal together.
```